DEPLOYING.md

```diff
@@ -1,62 +1,351 @@
+* Next, it will ask you for your location (time zone). Select the appropriate one and press `Continue`.
+
+![](https://i.imgur.com/g9pL3sX.png)
+
+* Choose your `User name`, `Password` and `log in` option.
+* Click on `Continue` to begin installation
 
+![](https://i.imgur.com/86zeDT9.png)
+
+* ... and wait to finish. When finished you will be asked to restart computer (in this case Ubuntu VM)
+
+![](https://i.imgur.com/IaOvbnD.png)
+
+* Click on `Restart Now`
+
+---
+---
+
+## 4. Additional settings for Ubuntu in VM
+
+#### 1. Git installation
+
+Open `Terminal` app, than run following command:
 ```
-sudo apt-get update
-sudo apt-get install git docker-compose build-essential
+sudo apt update
+sudo apt install git
 ```
 
-Then start and enable the docker service and verify operation
 
+#### 2. `docker-compose` installation
+
+Open `Terminal` app, than run following command:
 ```
-sudo systemctl start docker
-sudo systemctl enable docker
-sudo docker info
+sudo apt update
+sudo apt install docker-compose
 ```
 
-__Note:__ In case you need to use your local county mirrors as pip registry please set pip index URL:
+---
+---
+
+## 5. CCD Vital Strategies installation
 
+1. Clone repository from https://github.com/datopian/ckan-cloud-docker
 ```
-export PIP_INDEX_URL=<<https://local-mirror-url.xy>>
+git clone https://github.com/datopian/ckan-cloud-docker
 ```
 
+2. Change directory to `/ckan-cloud-docker` to enter the cloned folder.
+```
+cd ckan-cloud-docker
+```
 
-#### Extra dependencies
+3. Change branch to `ccd-windows-philippines`
+```
+git checkout ccd-windows-philippines
+```
 
-- You will also need a SMTP server and its credentials for CKAN to work properly. This will not obstacle deployment, CKAN will be up and running, but won't be able to send emails (e.g. on password reset). You will be asked to provide SMTP server credentials while running `make secret` script, see below.
+4. Run
+```
+python create_secrets.py
+```
 
-### Source files and configuration
+Hit `Enter` on every question.
 
-Now we have the runtime, next we need to download the cluster configuration files and build the services.
+5. Run
+```
+docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-philippines-theme.yaml up -d --build nginx
+```
 
-Navigate to where you want the source files to live on your server (e.g. `/opt`) and clone the repository:
+and wait setup to finish
 
+6. Run
 ```
-cd /opt
-git clone https://github.com/ViderumGlobal/ckan-cloud-docker.git
-cd ckan-cloud-docker
+docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-philippines-theme.yaml down
+``` 
+
+and wait to finish.
+
+7. In the folowing files change `CKAN_BRANCH:-ckan-2.7.3` to `CKAN_BRANCH:-ckan-2.8.1`:
+  * **`docker-compose.yaml`** - line #103
+  * **`.docker-compose.vital-strategies-philippines-theme.yaml`** - lines #24 and #43
+  * **`ckan/Dockerfile`** - line #61
```

Comment:
I'm still concerned about this part. Let's either make it 2.8.X from the begging or debug. What happens if you have 2.8.X all the time?